NAMESPACE

```diff
@@ -38,6 +38,7 @@ export(groupedLOCF)
 export(imputeColumn)
 export(initializeParamsForTesting)
 export(isBinary)
+export(nullPointer)
```

Comment:
If you pull in master, `isTargetYN` will end up here and your conflicts will go away.